R/input-dark-mode.R

```diff
@@ -0,0 +1,84 @@
+#' Dark mode input control
+#'
+#' Creates a button that toggles between dark and light modes, specifically for
+#' toggling between light and dark [Bootstrap color
+#' modes](https://getbootstrap.com/docs/5.3/customize/color-modes/) -- a new
+#' feature added in [Bootstrap
+#' 5.3](https://getbootstrap.com/docs/5.3/migration/#color-modes).
+#'
+#' @param ... Additional attributes to be passed to the input control UI, such
+#'   as `class`, `style`, etc.
+#'
+#'   In `toggle_dark_mode()`, the `...` are included for future extensibility
+#'   and are currently ignored.
+#' @param id An optional input id, required if you'd like to be able to
+#'   reactively read programmatically toggle the current color mode.
+#' @param mode The initial mode of the dark mode switch. By default or when set
+#'   to `NULL`, the user's system settings for preferred color scheme will be
+#'   used. Otherwise, set to `"light"` or `"dark"` to force a particular initial
+#'   mode.
+#' @inheritParams tooltip
```

Comment:
```suggestion
```

Reply:
This is for `session` (which comes in `toggle_dark_mode`). It'd be nice to just have one consistent place where we document `session` so we can inherit that param. I picked tooltip because this component has a similar origin...

Reply:
I moved the docs in here, we can handle the de-duplication separately

Reply:
Ah ok, sorry, just wanted to make sure it was there for a reason. I don't mind either way